test/devicecontroller/DefaultDeviceController.test.ts

```diff
@@ -464,6 +475,17 @@ describe('DefaultDeviceController', () => {
       await deviceController.chooseAudioOutputDevice(stringDeviceId);
       expect(spy.called).to.be.true;
     });
+
+    it('binds the audio device if the audio-video is bound with groupId', async () => {
```

Comment:
This new `chooseAudioOutputDevice` test is also not related to your fix for `chooseAudioInputDevice`. I added an overall comment for testing strategies.

Reply:
This is for just testing the mapping.